src/main/java/com/github/manolo8/darkbot/extensions/features/FeatureRegistry.java

```diff
@@ -119,8 +118,8 @@ private <T> Optional<T> getFeature(String id) {
                 feature.setInstance(instance = featureLoader.loadFeature(feature));
                 return Optional.of(instance);
             } catch (Throwable e) {
-                feature.getIssues().addFailure("bot.issue.feature.failed_to_load", IssueHandler.createDescription(e));
-                e.printStackTrace();
+                if (!feature.addIssue(e, "bot.issue.feature.failed_to_load", PluginIssue.Level.ERROR))
+                    e.printStackTrace();
```

Comment:
you shouldn't make printstacktrace conditional, always make the console show the error, it HAS happened, don't hide it away